src/commands/dsyms/__tests__/upload.test.ts

```diff
@@ -279,7 +279,7 @@ describe('execute', () => {
   test('Should succeed with folder input', async () => {
     const {context, code} = await runCLI('src/commands/dsyms/__tests__/fixtures/')
     const outputString = context.stdout.toString()
-    const output = outputString.split(EOL)
+    const output = outputString.split('\n')
```

Comment:
Wouldn't `\n` be less correct on Windows?

Reply:
I think it's only the case for **_file_** line-endings, and not for command line `stdout`/`stderr`. Or maybe Node.js changes them from `\r\n` to `\n`.

When running the following on Windows:
```sh
require('child_process').exec('git --help', (error, stdout, stderr) => console.log(JSON.stringify({stdout})))
```

You get (on both `cmd` and Powershell, so I don't think it's related to shell that is used): 

![image](https://github.com/user-attachments/assets/5fe25042-df4e-4b57-98fa-1ddd81d118b7)

And now that _we do_ run the tests in the CI on Windows, we know that `.split('\n')` works, while `.split(os.EOL)` doesn't work ([example of failing CI](https://github.com/DataDog/datadog-ci/actions/runs/14222643753/job/39854313102#step:5:4521)).